lib/delete.js

```diff
@@ -44,10 +44,10 @@ function removeScheduled(uri) {
     scheduled: false,
     scheduledTime: null
   }).then(function (resp) {
-    log('info', 'unscheduled page updated:', uri);
+    log('info', `unscheduled page updated: ${uri}`);
     return resp;
   }).catch(function (err) {
-    log('error', 'error update page list for unschedule', err.stack);
+    log('error', `error update page list for unschedule ${err.stack}`);
```

Comment:
does the stack format correctly if it's injected in as a string?

Reply:
It would, but you're right about the inconsistencies, I'm going to just pass the error through as a full object.